cmd/helm/status.go

```diff
@@ -116,6 +119,9 @@ func (s statusPrinter) WriteTable(out io.Writer) error {
 	fmt.Fprintf(out, "NAMESPACE: %s\n", s.release.Namespace)
 	fmt.Fprintf(out, "STATUS: %s\n", s.release.Info.Status.String())
 	fmt.Fprintf(out, "REVISION: %d\n", s.release.Version)
+	if s.showDescription && s.release.Info.Description != "" {
```

Comment:
I think the description should be shown regardless if empty or not. The reasoning is that someone might parse the output and expect it to be shown.

Reply:
I think the user should parse the `DESCRIPTION` like `NOTES`, it's optional. Well, maybe it should be optional in Helm 4? Currently it's a explicit flag...

Reply:
I think if `s.showDescription ` flag set then show the description even if empty

Reply:
Should it be optional in Helm 4? As you say, this flag will be removed in Helm 4. Do I need to add a TODO here?